tests/test-distribution/test-ee10-distribution/src/test/java/org/eclipse/jetty/ee10/tests/distribution/DisableUrlCacheTest.java

```diff
@@ -38,7 +40,9 @@
 @Isolated
 public class DisableUrlCacheTest extends AbstractJettyHomeTest
 {
-    @Test
+    private static final Logger LOG = LoggerFactory.getLogger(DisableUrlCacheTest.class);
+
+    @RepeatedTest(value = 40, failureThreshold = 1)
```

Comment:
good one.
we may need this one for other flaky tests?
maybe we do not need here as you fixed it with changing `awaitConsoleLogsFor`